Firestore/Example/Firestore.xcodeproj/xcshareddata/xcschemes/Firestore_FuzzTests_iOS.xcscheme

```diff
@@ -5,19 +5,6 @@
    <BuildAction
       parallelizeBuildables = "YES"
       buildImplicitDependencies = "YES">
-      <BuildActionEntries>
```

Comment:
Just curious, what's the reason for this removal?

Reply:
I think the default new scheme does not have the `<BuildActionEntries>` section. It is generated when the scheme build configurations change. But the default scheme is enough for us since it is testable. I thought to minimize this file if some section is not needed.